index.js

```diff
@@ -2,13 +2,23 @@
 
 type Queryable = Document | DocumentFragment | Element
 
+class InvariantError extends Error {
+  framesToPop: number
+
+  constructor(message) {
+    super(message)
+    this.name = 'Invariant Violation'
+    this.framesToPop = 1
+  }
+}
+
 export function closest<T: Element>(element: Element, selectors: string, type: Class<T>): T {
   const klass = type || HTMLElement
   const el = element.closest(selectors)
   if (el instanceof klass) {
     return el
   }
-  throw new Error(`Element not found: <${klass.name}> ${selectors}`)
+  throw new InvariantError(`Element not found: <${klass.name}> ${selectors}`)
```

Comment:
Does @github/web-systems have any opinions on tweaking these error messages to work better with @github's internal error reporting system (Haystack)?

Reply:
I would rather see them be called QueryError or something similar.